apollo-portal/src/main/resources/static/views/component/release-modal.html

```diff
@@ -51,103 +51,133 @@ <h4 class="modal-title" ng-show="toReleaseNamespace.isBranch && toReleaseNamespa
-                                    </td>
-                                    <td width="20%" title="{{config.masterReleaseValue}}">
-                                        <span ng-bind="config.masterReleaseValue"></span>
-                                    </td>
-                                    <td width="20%" title="{{config.oldValue}}">
-                                        <span ng-bind="config.oldValue"></span>
-                                    </td>
-                                    <td width="20%" title="{{config.newValue}}">
-                                        <span ng-bind="config.newValue"></span>
-                                    </td>
-                                    <td width="10%" ng-bind="config.item.dataChangeLastModifiedBy">
-                                    </td>
-                                    <td width="15%"
-                                        ng-bind="config.item.dataChangeLastModifiedTime | date: 'yyyy-MM-dd HH:mm:ss'">
-                                    </td>
-                                </tr>
+                            <tr ng-repeat="config in toReleaseNamespace.branchItems"
+                                ng-if="config.isModified || config.isDeleted">
+                                <td width="15%" title="{{config.item.key}}">
+                                    <span ng-bind="config.item.key"></span>
+                                    <span class="label label-danger"
+                                          ng-show="config.isDeleted">{{'Component.Publish.Deleted' | translate }}</span>
+                                </td>
+                                <td width="20%" title="{{config.masterReleaseValue}}">
+                                    <span ng-bind="config.masterReleaseValue"></span>
+                                </td>
+                                <td width="20%" title="{{config.oldValue}}">
+                                    <span ng-bind="config.oldValue"></span>
+                                </td>
+                                <td width="20%" title="{{config.newValue}}">
+                                    <span ng-bind="config.newValue"></span>
+                                </td>
+                                <td width="10%" ng-bind="config.item.dataChangeLastModifiedBy">
+                                </td>
+                                <td width="10%"
+                                    ng-bind="config.item.dataChangeLastModifiedTime | date: 'yyyy-MM-dd HH:mm:ss'">
+                                </td>
+                                <td width="5%">
+                                    <span>
+                                        <button type="button" class="btn btn-success"
+                                                ng-click="compare(config,0)">{{'Component.Publish.Text.Compare' | translate
```

Comment:
Is it possible to only have one compare button and the program parse the actual format? 
I don't think it's extensible if we need to add xml compare, yml compare in the future.

![image](https://user-images.githubusercontent.com/837658/76680249-f181f000-6621-11ea-98c6-128cf6fc7b16.png)

Reply:
what you say is correct,maybe because I use json more.

Reply:
I think Xml compare can use text compare model,maybe most people will use json cofiguartion in the text for applocation.For your word,maybe you want to support the xml comapre or other format,and you have a better position to place this configuration function?

Reply:
also can support all format,I will improve this after,and merge request this after I finish improved.Thank you very much.

Reply:
What I am proposing is to only show one button to compare, which is friendly to the user. Then for json format, we could use json compare mode, and for other formats, maybe we could just use text compare mode.